custom_components/ttlock/models.py

```diff
@@ -102,6 +102,13 @@ def _set_start_minute(cls, start_minute: int | None) -> int:
     def _set_end_minute(cls, end_minute: int | None) -> int:
         return end_minute or 0
 
+class AddPasscodeConfig(BaseModel):
+    """The passcode creation configuration."""
+
+    passcode: str = Field(None, alias="passcode")
+    passcode_name: str = Field(None, alias="passcodeName")
+    start_minute: int = Field(0, alias="startDate")
```

Comment:
Looking at the documentation, these fields are milliseconds since epoch. Could we use the `EpochMs` type here to handle the conversion between python datetime objects and what the api expects?

Reply:
I'd be happy to make it cleaner if only I knew how to do so ☺️
I'm not developer but just a hobbyist who tried to add a useful functionality.
It's my first time with Python and it was already a great performance for to make it work 😅
If you can make it better, do it and I'll be happy to help as far as I can.

Reply:
No worries, happy to help.
Could you test the code in https://github.com/jbergler/hass-ttlock/pull/81 and let me know if you see any issues?